src/Microsoft.ML.Transforms/HashJoiningTransform.cs

```diff
@@ -188,9 +187,9 @@ private static VersionInfo GetVersionInfo()
             IDataView input,
             string name,
             string source = null,
-             bool join = Defaults.Join,
+             bool join = Defaults.Combine,
```

Comment:
`bool join` => `bool combine`. And update the xml docs

Reply:
Actually, I am rethinking this comment. What's weird to me is that this is the "hash joining transform", and we are calling this behavior "combine". That seems inconsistent. We should use the same term to mean the same thing. Either always use "join" or always use "combine".

Reply:
I was actually thinking that the name `HashJoiningTransform` is not very informative. What do you think about `VectorHashingTransform`?

---
In reply to: [356308668](https://github.com/dotnet/machinelearning/pull/4514#discussion_r356308668) [](ancestors = 356308668)

Reply:
VectoHashingTransform implies each element is going to be hashed separately (which is the default behavior). With respect to @eerhardt's comment that we should user either Join or Combine consistently, a better name might be HashCombiningTransform.

---
In reply to: [356607566](https://github.com/dotnet/machinelearning/pull/4514#discussion_r356607566) [](ancestors = 356607566,356308668)

Reply:
I have reverted the changes in this file, since it is no longer being used in this PR. This transform is now not being used directly anywhere in the code, and just being kept around for backward compatibility of maml, and for its entry point.

---
In reply to: [427561266](https://github.com/dotnet/machinelearning/pull/4514#discussion_r427561266) [](ancestors = 427561266,356607566,356308668)